Buildings/Fluid/Storage/Plant/Validation/BaseClasses/PartialTankBranch.mo

```diff
@@ -0,0 +1,95 @@
+within Buildings.Fluid.Storage.Plant.Validation.BaseClasses;
+partial model PartialTankBranch "(Draft)"
+
+  package Medium = Buildings.Media.Water "Medium model";
+
+  parameter Modelica.Units.SI.AbsolutePressure p_CHWS_nominal=800000
+    "Nominal pressure of the CHW supply line";
+  parameter Modelica.Units.SI.AbsolutePressure p_CHWR_nominal=300000
+    "Nominal pressure of the CHW return line";
+  parameter Modelica.Units.SI.Temperature T_CHWS_nominal=7+273.15
+    "Nominal temperature of CHW supply";
+  parameter Modelica.Units.SI.Temperature T_CHWR_nominal=12+273.15
+    "Nominal temperature of CHW return";
+
```

Comment:
Minor/Optional: For those partial class of examples, it would actually be more relevant to expose the parameters of the component that you intend to vary in the different cases (namely: `allowRemoteCharging`). The parameters you have there are good if someone were to try out different values, although it is not critical since this class is specifically for the construction of the validation models.